pkgs/applications/audio/losslessaudiochecker/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl, autoPatchelfHook }:
+
+stdenv.mkDerivation {
+  pname = "losslessaudiochecker";
+  version = "2.0.5";
+
+  src = fetchurl {
+    url = "https://losslessaudiochecker.com/dl/LAC-Linux-64bit.tar.gz";
```

```suggestion
    url = "https://web.archive.org/web/20211119122205/https://losslessaudiochecker.com/dl/LAC-Linux-64bit.tar.gz";
```
We want to use versioned download links.